tensorflow/jobs/docker/2.4/py3/Dockerfile.gpu

```diff
@@ -11,17 +11,15 @@ RUN apt-get clean
 
 # Installing our custom python libraries
 RUN ${PIP} install --no-cache --upgrade \
-        amazon-braket-default-simulator==1.10.0 \
-        amazon-braket-ocean-plugin==1.0.9 \
-        amazon-braket-schemas==1.12.0 \
-        amazon-braket-pennylane-plugin==1.8.0 \
-        amazon-braket-sdk==1.33.0 \
+        amazon-braket-default-simulator==1.10.1 \
+        amazon-braket-schemas==1.13.1 \
+        amazon-braket-pennylane-plugin==1.9.0 \
+        amazon-braket-sdk==1.34.3 \
         awscli==1.23.7 \
         botocore==1.25.7 \
         boto3==1.22.7 \
         decorator==4.4.0 \
         dask==2.30.0 \
-        dwave-ocean-sdk==4.2.0 \
         h5py==3.2.1 \
         ipykernel==5.3.4 \
         jinja2==2.11.3 \
```

Comment:
Can update keras with what we have used for NBIs, if this doesn't conflict with tensorflow here.